server/data/incidentClient.ts

```diff
@@ -280,4 +282,36 @@ export default class IncidentClient {
       values: [formResponse, incidentDate, reportId],
     })
   }
+
+  async getReportInProgress(bookingId: number, status: string): Promise<inProgressReport> {
+    const result = await this.query({
+      text: `
+      select vr.id id, vr.incident_date as incidentdate
```

Comment:
Case is wrong here: `incidentDate`

Reply:
Yes, the problem I had was, that camel casing it in the select query response had no affect. It would still return everything in lower case